install/kubernetes/cilium/templates/cilium-agent/clusterrole.yaml

```diff
@@ -98,6 +98,8 @@ rules:
   - ciliumlocalredirectpolicies/finalizers
   - ciliumegressnatpolicies
   - ciliumendpointslices
+  - ciliumclusterwideenvoyconfigs
+  - ciliumenvoyconfigs
```

Comment:
I know this is current with verbs '*' but is currently being addressed in https://github.com/cilium/cilium/pull/19074

Please create add the specific verbs we will be using in Cilium's ClusterRole instead of `'*'`. Either make the change here or leave a comment in #19074 which verbs will be used for the ciliumenvoyconfigs.

Reply:
Changed to only include verbs `list` and `watch`.